surfactant/cmd/merge.py

```diff
@@ -20,17 +21,39 @@
     default="surfactant.output.cytrics_writer",
     help="SBOM output format, options=surfactant.output.[cytrics|csv|spdx]_writer",
 )
+@click.option(
+    "--input_format",
+    is_flag=False,
+    default="surfactant.input_readers.cytrics_reader",
+    help="SBOM input format, assumes that all input SBOMs being merged have the same format, options=surfactant.input_readers.[cytrics|cyclonedx|spdx]_reader",
+)
 @click.command("merge")
-def merge_command(input_sboms, sbom_outfile, config_file, output_format):
+def merge_command(input_sboms, sbom_outfile, config_file, output_format, input_format):
     """Merge two or more INPUT_SBOMS together into SBOM_OUTFILE.
 
     An optional CONFIG_FILE can be supplied to specify a root system entry
     """
     pm = get_plugin_manager()
     output_writer = pm.get_plugin(output_format)
+    input_reader = pm.get_plugin(input_format)
+    if input_reader is None:
+        for plugin in pm.get_plugins():
+            try:
+                if plugin.short_name().lower() == input_format.lower() and hasattr(
+                    plugin, "read_sbom"
+                ):
+                    input_reader = plugin
+                    break
+            except AttributeError:
+                pass
+
+    if input_reader is None:
+        logger.error(f'No output format "{input_format}" found')
```

Comment:
```suggestion
        logger.error(f'No input reader plugin for format "{input_format}" found')
```

Reply:
Note: This pattern of searching for input/output readers seems like something that is going to come up in several places -- may be useful to consider moving most of the code into a plugin manager helper function that searches for a plugin with the given plugin name and implementing a given hooked function name ("read_sbom").

Reply:
I have moved the code for finding input readers and output writers into a separate function in manager.py